flair/models/clustering/run_BIRCH.py

```diff
@@ -0,0 +1,17 @@
+from flair.data import Sentence
```

Comment:
the run scripts should be removed from the PR. You can put the scripts as instructions into the issue text.